workflow/ttlcontroller/ttlcontroller.go

```diff
@@ -187,9 +192,14 @@ func (c *Controller) deleteWorkflow(key string) error {
 
 		err = c.wfclientset.ArgoprojV1alpha1().Workflows(wf.Namespace).Delete(wf.Name, &metav1.DeleteOptions{PropagationPolicy: commonutil.GetDeletePropagation()})
 		if err != nil {
-			return err
+			if apierr.IsNotFound(err) {
```

Comment:
I've not seen this error apart from initial testing - prevented by line 155 I think, but a good bonus check